OneSignalSDK/onesignal/core/src/test/java/com/onesignal/core/internal/operations/OperationRepoTests.kt

```diff
@@ -590,6 +593,20 @@ class OperationRepoTests : FunSpec({
         // Then
         result shouldBe null
     }
+
+    test("IOperationRepoLoadedListener") {
+        val mocks = Mocks()
+        val mockIdentityModel = mockk<IdentityModel>()
+        val mockIdentityModelStore = mockk<IdentityModelStore>()
+        val recovery = RecoverFromDroppedLoginBug(mocks.operationRepo, mockIdentityModelStore, mocks.configModelStore)
+
+        mocks.operationRepo.start()
+        recovery.start()
+
+        verify {
+            mocks.operationRepo.subscribe(recovery)
+        }
+    }
```

Comment:
This is testing `RecoverFromDroppedLoginBug` instead of OperationRepo, as the only thing we are verifying is that `operationRepo.subscribe` was called. This is a fine test, but such a test would belong in `RecoverFromDroppedLoginBugTest.kt`.

If we want to test the new `OperationRepo` feature, we should test that it actually fired the event instead.

Reply:
Good point! I have fixed that and add one test unit in each class to ensure the event listener works correctly.